.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2.1
+
+orbs:
+  codacy: codacy/base@0.3.8
```

Comment:
Bump to `1.2.0` here

Reply:
You can use https://circleci.com/orbs/registry/?filterBy=all&query=codacy to check our orbs' versions

Reply:
Done